astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,427 @@
+                ("arcmodel", neptune_arcmodels[int(neptune_arcmodel)]),
+                ("extra_ra", ""),  # figure options below this line, can all be hardcoded and ignored
+                ("extra_ra_type", "hours"),
+                ("extra_dec", ""),
+                ("extra_name", ""),
+                ("title", ""),
+                ("labels", "Small (6 points)"),
+                ("moonpts", "0"),
+                ("blank", "No"),
+                ("opacity", "Transparent"),
+                ("peris", "None"),
+                ("peripts", "4"),
+                ("arcpts", "4"),
+                ("meridians", "Yes"),
+                ("output", "html"),
+            ]
+        )
+
+        # return request_payload if desired
+        if get_query_payload:
+            return request_payload
+
+        # query and parse
+        response = self._request(
+            "GET", self._url, params=request_payload, timeout=self.timeout, cache=cache
+        )
+
+        return response
+
+    def _parse_result(self, response, verbose=None):
+        """
+        Routine for parsing data from ring node
+
+        Parameters
+        ----------
+        self : RingNodeClass instance
+        response : list
+            raw response from server
+
+
+        Returns
+        -------
+        bodytable : `astropy.QTable`
+        ringtable : `astropy.QTable`
+        """
+        try:
+            self._last_query.remove_cache_file(self.cache_location)
+        except (FileNotFoundError, OSError):
+            # this is allowed: if `cache` was set to False, this
+            # won't be needed
+            pass
+
+        soup = BeautifulSoup(response.text, "html.parser")
+        text = soup.get_text()
+        textgroups = re.split(2*os.linesep+"|"+os.linesep+" "+os.linesep, text)  # need regex because some blank lines have spacebar and some do not
+        ringtable = None
+        for group in textgroups:
+            group = group.strip()
+
+            # input parameters. only thing needed is epoch
+            if group.startswith("Observation"):
+                epoch = group.splitlines()[0].split("e: ")[-1].strip()
+
+            # minor body table part 1
+            elif group.startswith("Body"):
+                group = "NAIF " + group  # fixing lack of header for NAIF ID
+                bodytable_names = ("NAIF ID", "Body", "RA", "Dec", "RA (deg)", "Dec (deg)", "dRA", "dDec")
+                bodytable_units = [None, None, None, None, u.deg, u.deg, u.arcsec, u.arcsec]
+                bodytable = table.QTable.read(group, format="ascii.fixed_width",
+                                        col_starts=(0, 4, 18, 35, 54, 68, 80, 91),
+                                        col_ends=(4, 18, 35, 54, 68, 80, 91, 102),
+                                        names=bodytable_names)
+                                        # units=(bodytable_units)) # this much cleaner way of adding units is supported in later versions but not in 3.7
```

Comment:
Are you sure about this? I would find it very strange if the Python version would have any effect on the keyword arguments `QTable.read()` can accept.

Reply:
The comment is wrong in the sense that it's not python 3.7 related, but oldest supported astropy related as the `units` kwarg got added in 4.1. 

We haven't really formalized the dropping of versions, but I kind of like to follow NEP29/SPEC0 with the caveat that for astroquery we do the drops more on need bases rather than automatically. Anyway, 24 months have passed since 4.1, so let's drop 4.0 support outside of this PR.